geode-core/src/main/java/org/apache/geode/internal/cache/wan/AbstractGatewaySender.java

```diff
@@ -1138,7 +1153,7 @@ int getTmpDroppedEventSize() {
   public void enqueueTempEvents() {
     if (this.eventProcessor != null) {// Fix for defect #47308
       // process tmpDroppedEvents
-      EntryEventImpl droppedEvent = null;
+      EntryEventImpl droppedEvent;
```

Comment:
this code change is unnecessary

Reply:
The original statement is redundant (as pointed by the IDE) as the variable will be reassigned in the next statement (while) 
anyway.